source/_integrations/roon.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: RoonLabs Media Player
+description: Instructions on how to integrate a RoonLabs multi room player into Home Assistant.
+ha_category:
+  - Media Player
+ha_release: 0.114
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@pavoni'
+ha_domain: roon
+---
+
+The Roon integration allows you to control [RoonLabs](https://roonlabs.com/) music players from Home Assistant.
+
+This integration uses Roon Core, a Roon application that runs on a machine on your network. Via Roon Core, Home Assistant can control all the Roon music players on your network.
+
+To integrate with Roon, you need to provide Home Assistant with the Hostname or IP address of the machine that runs your Roon Core, and then authorize Home Assistant in the Roon application.
+
+<div class='note'>
+
+If you use an IP address please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
+
+</div>
+
+### Configuration
+
+<div class='note'>
+
+  You need the Hostname or IP adresss of the machine that runs your Roon Core. This might be a machine name (which can be followed buy .local, eg myserver.local) or can be an IP address.
+
+</div>
```

Comment:
Do we really need 2 note blocks? I think this can just be part of the normal text.